programs/svm-spoke/src/state/fill.rs

```diff
@@ -13,3 +13,19 @@ pub struct FillStatusAccount {
     pub status: FillStatus,
     pub relayer: Pubkey,
 }
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone)]
+pub struct V3RelayData {
```

Comment:
maybe better to have this under something like `src/common/v3_relay_data`? then we could also move `RefundAccount` to `src/common/refund_account` as it also does not strictly belong to the state

Reply:
ye, I think that's better. it's not really part of fill, and it's also not "state" in that it's not initialized. I'll update the way you've recommended.

I did not pull `refund_account`, though, as I think we might still end up changing this structure given the conversation we had last week where by we don't let the account be either TokenAccount OR ClaimAccount but rather pass in both of them to prevent relayers front running this set of methods.